README.md

```diff
@@ -0,0 +1,47 @@
+# Solution_01 Answers to Assignment Part 1
+
+a) The command 'git status' does output the status of the working directory and the staging area (eg which files are changed).
+
+b) In SwiftUI, anything that gets drawn to a screen, is a view.
+
+c) print("Hello world") is an example of a global function call. viewModel.getData() is an example of an instance method call.
+
+d) Some Views: Text, Button, Slider, Image
+
+e) To create a new local repository: git init project-directory
+
+f) To Preview an app in multiple orentations: ContentView().previewInterfaceOrientation(.portrait) ContentView().previewInterfaceOrientation(.landscapeLeft) ...
+
+g) An App ist made up of a combination of classes and structs that contain properties and methods.
+
+h) Two components of a SwiftUI Button: title, action
+
+i) A local repository sits at the local machine, while a remote repository sits at a remote server (eg at github).
+
+j) Camel Case: anInterestingLongVariableName
+
+k) To create a bracnch: git branch my-new-branch Branches are used to develop specific features separate from each other
```

Comment:
You can also use `git checkout -b my-new-branch` if you want to create the branch and immediately switch into it.